languages/python/sqlalchemy-oso/tests/test_roles2.py

```diff
@@ -2382,6 +2382,14 @@ def test_roles(init_oso, auth_sessionmaker, User, Organization, Repository, Issu
     assert demo_repo.id in result_ids
     assert ios.id not in result_ids
 
+    oso.actor = leina
+    oso.checked_permissions = {Issue: "edit"}
+    auth_session = auth_sessionmaker()
+
+    results = auth_session.query(Issue).all()
+    assert len(results) == 1
+    result_ids = [repo.id for repo in results]
```

Comment:
The results are issues, not repos.
```suggestion
    result_ids = [issue.id for issue in results]
```